src/auxiliary.jl

```diff
@@ -69,7 +69,12 @@ end
 Return the coefficient of order `n::Int` of a `a::Taylor1` polynomial.
 """
 get_coeff(a::Taylor1, n::Int) = (@assert 0 ≤ n ≤ a.order+1;
-    return a.coeffs[n+1])
+    return a[n+1])
+
+getindex(a::Taylor1, n::Int) = a.coeffs[n]
+getindex(a::Taylor1, n::UnitRange) = a.coeffs[n]
```

Comment:
Yes we do. I wanted to begin just with Taylor1 to get it right. I won't be able to get to the rest for a while if you want to have a go.